app/Http/Requests/Auth/ForgetPassword.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Requests\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ForgetPassword extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
```

Comment:
Is this comments useful for us because it is same in all the request files.

Reply:
@kajal-7span Done

Reply:
Already done by @harshil-7span